packages/server/src/controllers/bookings.ts

```diff
@@ -49,20 +51,48 @@ export const createBooking = async (ctx: Context) => {
 export const getBooking = async (ctx: Context) => {
   const { bookingId } = ctx.params;
   try {
-    const booking = await getBookingById(bookingId);
+    const booking = await retrieveBookingById(bookingId);
     ctx.body = booking;
   } catch (error) {
     console.log(`Error while retrieving a booking: ${bookingId}`);
     ctx.status = 500;
   }
 };
 
+export const getBookingsByInterval = async (ctx: Context) => {
```

Comment:
I don't understand the `get..` and `retrieve..` namings, shouldn't we stick to a single naming convention?
I vote for `getEntit(y,ies)ByField` for the methods that could throw if the entity is not found and `findEntit(y,ies)ByField` for methods which returns `Entity | null`

Reply:
See above :))

Reply:
we could suffix all request handlers by `handler`. Like: `getAllBookingsHandler` or `signupHandler`, WDYT ?

Reply:
Yes could do, but it's increasing the length of each name. 
I check to see if there is some conventions on this matter :))